src/components/Document/index.js

```diff
@@ -9,12 +9,10 @@ DocumentNotFound.defaultProps = {
   message: 'We are unable to detect an identity document in this image. Please try again.'
 }
 
-export const DocumentTitle = ({ useCapture, captureTitle, uploadTitle }) =>
-  <div className={theme.title}>{useCapture ? captureTitle : uploadTitle}</div>
-
-DocumentTitle.defaultProps = {
-  captureTitle: 'Place your document in the rectangle',
-  uploadTitle: 'Upload a picture of your document'
+export const DocumentTitle = ({ useCapture, side }) => {
```

Comment:
having the static text as props allows us to be closer to the possibility of allowing full text customization.